### PR TITLE
Iode to ephemeris

### PR DESCRIPTION
**Pull request by denniszollo**
We were missing a field in the SBP definition of the ephemeris

### CODE REVIEW

File: spec/tests/yaml/swiftnav/sbp/test_observation.yaml [hunk 1]
```diff
@@ -349,305 +349,5 @@ tests:
     payload: KKNEGC4HEIRr2XkOlP//ASyvkwCBQsh9lB3+/5kpDrEDgLL4iI9C/v/eEriFDZ41soYqGwIA7R4hYBYmU1Z2qG///y040Fge
     preamble: '0x55'
     sender: '0x4c3'
```
Comment (mookerji) [inline spec/tests/yaml/swiftnav/sbp/test_observation.yaml]: What happened to the ephemeris unit test case here?

**Comment by mookerji** [comment]
Question about the tests. I would also check with @fnoble .

File: spec/tests/yaml/swiftnav/sbp/test_observation.yaml (hunk 1 above)
Comment (denniszollo) [inline spec/tests/yaml/swiftnav/sbp/test_observation.yaml]: oh weird.  the log I used to generate the unit test must have had only one type of ephemeris message in it.  I'll regenerate the unit tests with a log with both types of ephemeris.  Thanks for pointing that out.

**Comment by fnoble** [comment]
lgtm once unit tests are restored

**Comment by denniszollo** [comment]
Ephemeris tests back in with the correct field.  The old ephemeris message is checked in the new "deprecated" yaml per @mfine's updates.  I think this is ready to go.

**Pull request merged**